packages/rlc-common/src/metadata/buildReadmeFile.ts

```diff
@@ -169,6 +169,7 @@ const credential = new InteractiveBrowserCredential({
   tenantId: "<YOUR_TENANT_ID>",
   clientId: "<YOUR_CLIENT_ID>"
  });
+const subscriptionId = "00000000-0000-0000-0000-000000000000";
```

Comment:
Pls include the hlc change in this pr also.

Reply:
Pls also consider the issue https://github.com/Azure/autorest.typescript/issues/3162